README.md

```diff
@@ -1,2 +1,48 @@
-# PairStormIDE
-Pair Storm IDE. The project, created by group Lv-425.C++ of the Softserve IT Academy.
+
+
+:cloud: Pair Storm IDE :cloud:
+==============================
+
+[![Code Style](https://img.shields.io/badge/Code%20Style-Qt%20Coding%20Style-green)](https://wiki.qt.io/Qt_Coding_Style) 
+![Contributions](https://img.shields.io/badge/Contributions-Welcome-brightgreen)
+[![License](https://img.shields.io/badge/License-MIT-lightgrey)](/LICENSE) 
+[![Slack](https://img.shields.io/badge/Slack-Join-red)](http://cppteamsoftserve.slack.com/)
+
+
+<img src="src/img/SPLASHSCREEN.png" alt="PairStorm" align="center">
+
+Pair Storm is a project oriented integrated development environment, which based on conception of pair programming. The software contains many useful features, which aim to help programmers to write good and clearly code. About them you can read below. Also Pair Storm helps to organize coworking between two or more programmer. To reaching these goal in the program were added many modules, which help to communicate, share code etc. IDE was created by group LV-425.C++ of the SoftServe IT Academy. 
+
+:two_men_holding_hands: User-Friendly Interface :two_men_holding_hands:
+======================================================================
+<img align="right" height=160 width=410 style="margin-left: 20px" src="https://user-images.githubusercontent.com/52821952/64379078-d01ec680-d036-11e9-8f44-14010ae4b005.png">
+Interface of Pair Storm is pretty simple. If you were using integrated development environment like Visual Studio, Qt Creator, CLion and etc before, work with Pair Storm will not be a problem for you. The most used tools is always in sight. If you want access to advanced settings use the relevant menu.
+ 
+:speech_balloon: Embedded Chat :speech_balloon:
+===============================================
+<img align="left" width=207 style="margin-right: 20px" src="https://user-images.githubusercontent.com/52821952/64411882-dcd30700-d096-11e9-9460-9deff5e4c14c.png">
+Pair Storm IDE has embedded chat. It helps to communicate with another programmers during development process. In chat can hang out many users at the same time. All messages are saving in local database, which is unique for each project. Embedded chat is secure. Get access to the message history can only authorized users, which work have appropriate premissions.
+
+:hammer: Refactoring Tools :hammer:
+===================================
+<img align="right" width=220 style="margin-left: 20px" src="https://user-images.githubusercontent.com/52821952/64385117-dc0e8680-d03e-11e9-993a-bc40d504a335.png">
+IDE gives users many useful tools, which make development process easier and faster. For example, Pair Storm has embedded class generator. Using it programmer can create .h and .cpp files, define new methods in both file and etc. 
+
+Also Pair Storm have many useful features, which help programmer to write good code: auto tabulation, auto completion and etc.
+
```

Comment:
IDE has many useful tools, which make development process easier and faster. For example, Pair Storm has embedded class generation. Thus user can easily create corresponding .h and .cpp files along with defining new methods there.

Meanwhile, Pair Storm has a bunch of useful features that help programmer to write clean code: auto-tabulation, auto-completion, etc.